apps/addons/models.py

```diff
@@ -624,12 +624,15 @@ def share_url(self):
 
     @property
     def automated_signing(self):
-        # We allow automated signing for add-ons which are not listed, and
-        # have not asked for side-loading privileges (full review).
+        # We allow automated signing for add-ons which are not listed.
         # Beta versions are a special case for listed add-ons, and are dealt
         # with on a file-by-file basis.
-        return not (self.is_listed or
-                    self.status in (amo.STATUS_NOMINATED, amo.STATUS_PUBLIC))
```

Comment:
I would like confirmation on that. There seems to be contradicting information in https://blog.mozilla.org/addons/2015/04/15/the-case-for-extension-signing/ (see "the adware scourge" paragraph, and `This is not a concern about hypotheticals, we have many documented cases of add-ons disabling the mechanisms through which we inform users and give them control over their add-ons.`

Ping @jvillalobos